### ISSUE TITLE
Record Frames in Remote Desktop and Remote Webcam

### ISSUE THREAD

**Issue by kernelzr** [quasar/QuasarRAT#512]
This RAT is fantastic, without too many unnecessary functions, thanks for sharing.

You should implement an option to save the frames obtained as image, these features is very important a RAT.

**Comment by mshafiqsb** [comment]
I have done this , how i upload it?

**Comment by MaxXor** [comment]
@mshafiqsb See here: https://github.com/quasar/QuasarRAT/blob/master/CONTRIBUTING.md

**Comment by mshafiqsb** [comment]
i did

**Comment by MaxXor** [comment]
@mshafiqsb No, that's wrong. You completely replaced the files. Please only commit the changes and then create a pull request.

**Comment by mshafiqsb** [comment]
i did it again, now check the pull request. 
i replace the files because, there are some design changes, i remove panel, create menu bar and place all control there. In this way i save space and give a better/standard look.
